utils/generate_types/index.js

```diff
@@ -93,7 +93,7 @@ class TypesGenerator {
 
     let overrides = await parseOverrides(overridesFile, className => {
       const docClass = this.docClassForName(className);
-      if (!docClass)
+      if (!docClass || !this.shouldGenerate(className))
```

Comment:
I think this is fine, but double check with @dgozman

Reply:
It affected the output in the expected manner, so should be good. We commit the results of the run.